Exiled.API/Extensions/CommonExtensions.cs

```diff
@@ -17,13 +17,18 @@ namespace Exiled.API.Extensions
     /// </summary>
     public static class CommonExtensions
     {
+        /// <summary>
+        /// Gets the initialized global random class.
+        /// </summary>
+        public static System.Random Random { get; } = new();
```

Comment:
Use the one from Exiled.Loader

Reply:
![image](https://github.com/Exiled-Team/EXILED/assets/69786695/50364c74-eedc-4718-8299-7314913f8e6e)
When adding Exiled.Loader to Exiled.API ☠️

Reply:
oh yeah that normal i haven't thinks to that when i have said it